CHANGELOG.txt

```diff
@@ -14,6 +14,9 @@
 * Removed frontend integration tests (written with Selenium)
 * Provides the ability to declare cache expiration periods on a per-plugin basis
 * Minor UI improvements
+* Adds new setting CMS_INTERNAL_IPS for defining a set of IP address for which
+  the toolbar will appear for authorized users. If left unset, does nothing.
+
```

Comment:
Shouldn't "If left unset, does nothing" be better explained as "If left unset, every IP address is authorized"?